components/gif_picker/components/SearchGrid/SearchGrid.jsx

```diff
@@ -37,7 +40,7 @@ const mapDispatchToProps = ({
 const getStyle = makeStyleFromTheme((theme) => {
     return {
         background: {
-            backgroundColor: changeOpacity(theme.centerChannelColor, 0.05),
+            backgroundColor: changeOpacity(theme.centerChannelColor, 0),
```

Comment:
Since this colour is just going to be transparent anyway, could the code using `getStyle` be removed? As long as the div with this background looks fine without it, we could remove `getStyle` and the code in the render method that calls it.

Reply:
@hmhealey By the way I forgot to mention why I made this change. I made this change because emoji picker didnt have the same opacity setting for its background so they looked different. If this is not okay I can revert it. Or If its okay I'll remove the redundant style code blocks.

Categories tab has the same styling that makes the background slightly lighter. I'll be removing them aswell so that they look the same.

Reply:
I didn't notice we did that. That seems unintentional. I think it'd be better to be consistent between the two, but I'll defer to someone from UX.

@matthewbirtch How would you feel if we made the background colours of the emoji and gif picker the same then? Right now, the gif picker is slightly lighter/darker than the emoji picker which just uses the centre channel colour for its background.

Here's how it looks currently on master:
https://user-images.githubusercontent.com/3277310/108428177-2a501f00-720c-11eb-94c1-27fa13fb829e.mov

Reply:
> @matthewbirtch How would you feel if we made the background colours of the emoji and gif picker the same then? Right now, the gif picker is slightly lighter/darker than the emoji picker which just uses the centre channel colour for its background.

Totally fine with it being the same bg color (just using the centre channel bg color).